docker/opg-s3-antivirus-update/Dockerfile

```diff
@@ -17,24 +11,13 @@ COPY . .
 
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/main ./cmd/opg-s3-antivirus-update
 
-FROM lambci/lambda:go1.x
+FROM alpine:3.15
 
-ENV DOCKER_LAMBDA_WATCH=1
-ENV DOCKER_LAMBDA_STAY_OPEN=1
-ENV AWS_LAMBDA_FUNCTION_HANDLER=main
-ENV LD_LIBRARY_PATH=/var/task/lib
+RUN apk update && apk add --no-cache clamav
 
-COPY ./freshclam.conf /etc
-COPY --from=build-clamav /usr/bin/freshclam /var/task/bin/freshclam
-COPY --from=build-clamav /usr/lib64/libclam* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libfreshclam* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libgnutls* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libhogweed* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libjson* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libltdl* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libnettle* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libpcre* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libprelude* /var/task/lib/
-COPY --from=build-clamav /usr/lib64/libtasn1* /var/task/lib/
+RUN mkdir -p /tmp/clamav && chown -R clamav:clamav /tmp/clamav
```

Comment:
`/tmp` is overwritten by Lambda in production but this is necessary locally, effectively emulating that behaviour